app/src/androidTest/java/com/example/henzoshimada/feeltrip/FeelTripTests.java

```diff
@@ -1,11 +1,11 @@
 package com.example.henzoshimada.feeltrip;
 
-import android.location.Location;
```

Comment:
no, it is Location would be two Doubles, longitude and latitude. But elastic search only support float, so we might have some issues on accuracy